lib/zenaton/services/graph_ql/create_workflow_schedule_mutation.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'zenaton/services/graph_ql/base_mutation'
+require 'zenaton/workflows/version'
+
+module Zenaton
+  module Services
+    module GraphQL
+      # Mutation parameters for scheduling a Task
+      class CreateWorkflowScheduleMutation < BaseMutation
+        def initialize(workflow, cron, app_env)
+          super
+          @workflow = workflow
+          @cron = cron
+          @app_env = app_env
+        end
+
+        # The body of the GraphQL request
+        def body
+          { 'query' => query, 'variables' => variables }
+        end
+
+        # The query to be executed
+        def raw_query
+          <<~GQL
+            mutation ($input: CreateWorkflowScheduleInput!) {
+              createWorkflowSchedule(input: $input) {
+                schedule {
+                  id
+                }
+              }
+            }
+          GQL
+        end
+
+        # The variables used in the query
+        def variables
+          { 'input' => input }
+        end
+
+        private
+
+        def workflow_name
```

Comment:
It seems that this logic is duplicated

Reply:
Definitely. I'm not a fan of having this logic duplicated. Currently this method is in:
- `create_workflow_schedule_mutation.rb`
- `dispatch_workflow_mutation.rb`

I'd wait for a 3rd occurrence to have a better idea where to put this. I agree this is not the best place for it.